.github/workflows/wave.yml

```diff
@@ -52,12 +52,12 @@ jobs:
     # if: github.repository == 'nf-core/modules'
     if: ${{ needs.generate-matrix.outputs.conda-matrix != '[]' }}
     needs: generate-matrix
-    name: Build ${{ matrix.profile }} ${{ matrix.platform }} Container
+    name: 🌊 ${{ matrix.files }} ${{ matrix.profile }} ${{ matrix.platform }}
```

Comment:
🙅🏻 makes it difficult for newcomers to understand what is happening

Reply:
Yeah, I can agree with that. 

I was trying to clean up the names because it's also hard for newcomers and me to figure out what's happening when the name is cut off 🙃 

Open to suggestions 🤷🏻‍♂️ 

<img width="346" alt="image" src="https://github.com/user-attachments/assets/a45ad57e-0824-4e32-a0ca-3f8fd5a9a274">